### ISSUE TITLE
How to take screenshot of hidden windows

### ISSUE THREAD

**Issue by Pololot64** [python-xlib/python-xlib#163]
Hello Everyone! I have been digging around in Xlib for a couple of months and have figured many things out on my own but when it comes to compositing, I am lost. I know I need to be able to redirect the windows to a new pixmap but where would I access this afterwards? I need a code sample because I really cannot translate the docs for other languages to python. I know there are other closed issues but none actually gave any answers.

**Comment by Pololot64** [comment]
Should I just close this "issue" I think everyone is too busy

**Comment by vasily-v-ryabov** [comment]
Yes, no chance to dig into it right now. But let's keep it open. Someone may find time in the future.

**Comment by Pololot64** [comment]
Sounds good!
